### DISCUSSION TITLE
Change Poison Form title font

### DISCUSSION THREAD

**Discussion by behroozbc** [Taiizor/ReaLTaiizor#84] [Q&A] [answered]
Hi,
I want to change the Poison Form title(added an image to the example) font size and font style. However, I change the Appearence> Font does not affect on Form Title.
![image](https://user-images.githubusercontent.com/20381616/201844757-aed1d30c-fdf5-4917-a930-35b1fa68839b.png)

**Comment by Taiizor** [comment, marked as answer]
Some themes are built on their own unique fonts. For this reason, the font and font size cannot be changed. The Poison theme usually uses the Segoe UI text family. Changing this at this stage means rebuilding the poison theme. This is unfortunately not possible.

**Reply by behroozbc** [reply]
Thanks, Limiting font to the Segoe UI makes it impossible to design in my native language (Persian) or other non-English languages, because those languages use different fonts such as [Vazirmatn](https://github.com/rastikerdar/vazirmatn). If possible, please, add an option to change the font.

**Reply by Taiizor** [reply]
Frankly, I don't recommend using WinForms when there is such a thing as WPF. Nowadays, the use of WinForms is about to end. Therefore, I am not actively developing this library. I only dwell on the little things I see. Making such a change would break the structure of the theme. And I don't have that much time to devote to that. Sorry..

**Reply by behroozbc** [reply]
Thanks, I have the same idea but I could not find a good UI library.
Do you know a WPF UI library that has themes like Poison or Crown?

**Reply by Taiizor** [reply]
[WPFUI](https://github.com/lepoco/wpfui)
[PanuonUI](https://github.com/Mochengvia/PanuonUI.Silver)
[HandyControl](https://github.com/HandyOrg/HandyControl)
[MaterialDesign](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit)

**Reply by Taiizor** [reply]
They're not the style you want, but they have a more mdoern look.

**Reply by behroozbc** [reply]
I going to check those.
Thanks
